platforms/core-runtime/build-configuration/src/main/java/org/gradle/internal/buildconfiguration/tasks/DaemonJvmPropertiesModifier.java

```diff
@@ -45,6 +45,7 @@ public void updateJvmCriteria(
         File propertiesFile,
         JavaLanguageVersion toolchainVersion,
         @Nullable String toolchainVendor,
+        Boolean nativeImageCapable,
```

Comment:
:x: Same question here, if this isn't `@Nullable` why do we need `Boolean`?

Reply:
Indeed, that is probably what IJ generated when I added the argument. I should have reviewed it after.